src/app.py

```diff
@@ -26,74 +36,56 @@ def get_secret_value(secret_arn: str):
     return response.get('SecretString')
 
 
-def sleep_data(garth_client: Client) -> dict:
+def sleep_data(client: GarthClient) -> dict:
     """
     Retrieves the daily sleep data for the user associated with the provided Garmin client.
 
     Args:
-        garth_client (Client): The Garmin Connect API client.
+        client (GarthClient): The Garmin Connect API client.
```

Comment:
renaming these as moving the client to the global scope can result in this function reaching the global client directly rather than the client on the argument